src/distPlus/distribution/AlgebraicShapeCombination.re

```diff
@@ -245,9 +245,10 @@ let toDiscretePointMassesFromDiscrete = (s: DistTypes.xyShape): pointMassesWithM
   {n, masses, means, variances};
 };
 
-let combineShapesContinuousDiscreteAdd =
+let combineShapesContinuousDiscrete =
```

Comment:
If s1 is continuous and s2 is discrete, this detail matters a lot, so I'd suggest calling them thus. (continuousDistribution: DistTypes.xyShapes, discreteDistribution: DistTypes.xyShapes)...

You could also make an alias for xyShape, like ``type continuousXyShape = xyShape`` for specificity.